weejson/jackson/src/com/rallyhealth/weejson/v0/jackson/JsonGeneratorVisitor.scala

```diff
@@ -0,0 +1,127 @@
+package com.rallyhealth.weejson.v0.jackson
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.rallyhealth.weejson.v0.JsVisitor
+import com.rallyhealth.weepickle.v0.core.{ArrVisitor, ObjVisitor, SimpleVisitor, Visitor}
+
+/**
+  * Adapter to Jackson's [[JsonGenerator]] classes, used to serialize jvm objects to bytes.
+  *
+  * Allows WeePickle to emit all of jackson's supported serialization formats:
+  * - JsonGenerator
```

Comment:
```suggestion
  * - [[JsonGenerator]]
```
It looks like a class name.